package-overrides/github/witoldsz/angular-http-auth@1.0.0.json

```diff
@@ -0,0 +1,15 @@
+{
+  "directories": {
+    "lib": "src"
+  },
+  "main": "http-auth-interceptor",
+  "dependencies": {
+    "angular": "^1.0.0 <1.2.0"
```

Comment:
Arbitrary version ranges are not supported unfortunately. Rather do `~1.1.0` perhaps.